dapp/package.json

```diff
@@ -28,13 +28,13 @@
         "@boxfish-studio/react-cookie-manager": "^1.0.5",
         "@growthbook/growthbook": "^1.6.1",
         "@growthbook/growthbook-react": "^1.6.1",
-        "@iota/apps-ui-icons": "^0.4.1",
-        "@iota/apps-ui-kit": "^0.7.2",
-        "@iota/dapp-kit": "^0.5.3",
-        "@iota/graphql-transport": "^0.11.0",
+        "@iota/apps-ui-icons": "0.0.0-experimental-20251127115223",
+        "@iota/apps-ui-kit": "0.0.0-experimental-20251127115223",
+        "@iota/dapp-kit": "0.0.0-experimental-20251127115223",
```

Comment:
I guess this need to be reverted. Just bump dapp-kit version

Reply:
bumped necessary versions and rolled back some others: https://github.com/iotaledger/iota-names/pull/983/commits/147fda3e8dec398bc8002c20357ea06fb566039c